skins/cat17/src/scripts/main.js

```diff
@@ -1,7 +1,6 @@
 (function ($) {
 
     var init = function () {
-        smoothScroll();
```

Comment:
Please add a note in membershipForm, lest we forget.

Reply:
Done in e72fed42f44f9fd15d752ca09fcaa6be2b1073c9